base.mako

```diff
@@ -68,5 +68,35 @@
             ga('create', 'UA-104682927-1', 'auto');
             ga('send', 'pageview');
         </script>
+        <script>
+(function () {
+  function copyTextToClipboard(text) {
+    var textArea = document.createElement("textarea");
+    textArea.value = text;
+    textArea.style.top = "0";
+    textArea.style.left = "0";
+    textArea.style.position = "fixed";
+    document.body.appendChild(textArea);
+    textArea.focus();
+    textArea.select();
+    document.execCommand('copy');
+    document.body.removeChild(textArea);
+  }
+  var codeBlockElements = document.querySelectorAll('div#advanced ul')[8].getElementsByClassName("highlight");
+  for (let block of codeBlockElements) {
+        var btn = document.createElement("img");
+        btn.setAttribute("src", "./copy-icon.svg");
+        btn.setAttribute("alt", "copy");
+        btn.setAttribute("title", "Copy code");
+        btn.setAttribute("width", "16");
+        btn.setAttribute("height", "16");
+        block.insertBefore(btn, block.children[0]);
+        btn.addEventListener("click", function() {
+          var text = block.getElementsByTagName('pre')[0].innerText;
+          copyTextToClipboard(text);
+        })
+  }
+})()
```

Comment:
this script should live in the assets folder, the indentation here is also inconsistent

Reply:
a new folder with name "assets" is created and the script is moved there